server/vote_endpoints.go

```diff
@@ -15,14 +15,40 @@
 package server
 
 import (
-	"net/http"
-
+	"encoding/json"
 	"github.com/rs/zerolog/log"
+	"io"
+	"net/http"
 
 	"github.com/RedHatInsights/insights-operator-utils/responses"
 	"github.com/RedHatInsights/insights-results-aggregator/types"
 )
 
+// readFeedbackRequestBody likes the rule for current user
+func (server *HTTPServer) readFeedbackRequestBody(writer http.ResponseWriter, request *http.Request) (types.FeedbackRequest, bool) {
+	var feedback types.FeedbackRequest
+
+	err := json.NewDecoder(request.Body).Decode(&feedback)
+	switch {
+	case err == io.EOF:
+		feedback.Message = ""
+		return feedback, true
+	case err != nil:
+		handleServerError(writer, err)
+		return feedback, false
+	}
+
+	if len(feedback.Message) > 250 {
+		handleServerError(writer, &types.ValidationError{
+			ErrString:  "String is longer then 250",
+			ParamName:  "message",
+			ParamValue: feedback.Message,
```

Comment:
Maybe it would be better to take the first 250 characters of the message

Reply:
If the front-end doesn't officially allow >250 characters to be submitted, then it may be okay to throw it away. Otherwise, we should probably get the first 250 characters and somehow indicate that the rest of it has been truncated.

Reply:
Yep, agree, truncate is not very good idea. Much better just return error.

Reply:
@bond95 what if the user send, say, 1MB of the data in message?